packages/fuels/tests/types.rs

```diff
@@ -1316,11 +1316,12 @@ async fn test_vector() -> Result<(), Error> {
         let arg = SomeStruct { a: vec![0, 1, 2] };
         methods.vec_in_struct(arg.clone()).call().await?;
     }
-    {
-        // array in vec
-        let arg = vec![[0u64, 1u64], [0u64, 1u64]];
-        methods.array_in_vec(arg.clone()).call().await?;
-    }
+    // TODO: It fails, I don't know why
+    // {
+    //     // array in vec
+    //     let arg = vec![[0u64, 1u64], [0u64, 1u64]];
+    //     methods.array_in_vec(arg.clone()).call().await?;
```

Comment:
This call fails, I'm investigating it, but maybe someone has ideas=)

Reply:
cc @digorithm @mohammadfawaz 

this is also broken between fuel-rs and sway on their respective master branches and appears to be unrelated to the fuel-core upgrade

Reply:
https://github.com/FuelLabs/sway/issues/3188